pgbouncer/test/test_pgbouncer.py

```diff
@@ -22,6 +23,9 @@ class TestPgbouncer(AgentCheckTest):
     CHECK_NAME = 'pgbouncer'
 
     def test_checks(self):
+        pgbouncer_version = os.environ.get('FLAVOR_VERSION', 'latest')
+        pgbouncer_deprecated = pgbouncer_version in ('1.5', '1.7')
```

Comment:
Thanks for writing the tests! I'm not sure we want to deprecate older versions of PGBouncer. Could we rename this to something like 'pgbouncer_pre18` or something similar here?

Reply:
Good idea! I was trying to use something other than "old" and "new" 😅